src/index.ts

```diff
@@ -40,9 +40,7 @@ export const beautifier: Beautifier = {
     return new Promise<string>((resolve, reject) => {
       let lang = "auto";
       switch (data.language.name) {
-        case "CSV":
-          lang = "csv";
-          break;
+        // case "CSV":   lang = "csv";   break;
```

Comment:
I think this can be removed instead of commented out. Original code looks simple enough 👍